lib/active_job_status.rb

```diff
@@ -18,5 +21,9 @@ def fetch(job_id)
       status = store.fetch(job_id)
       JobStatus.new(status)
     end
+
+    def store
+      @store or raise NoStoreError, "can't use ActiveJobStatus without store"
```

Comment:
Thanks!
I feel this is a bit hard to read:

```ruby
@store || raise(NoStoreError, "can't use ActiveJobStatus without store")
```

So I add a change like this:

```ruby
def store
  unless @store
    raise NoStoreError, "can't use ActiveJobStatus without store"
  end
  @store
end
```

(I will use `||` if you prefer to use it, rather than `unless` 👍 )